packages/vc-handler-ld-local/README.md

```diff
@@ -50,7 +50,165 @@ There are other utility scripts that help with development.
 
 - `yarn fix` - runs `eslint --fix` as well as `prettier` to fix code style.
 
-### Request examples:
+### Agent setup
+
+```typescript
+constructor(connection: Promise<Connection>) {
+    const privateKeyStore = new PrivateKeyStore(connection, new SecretBox(KMS_SECRET_KEY))
+    this._agent = createAgent<
+      ICredentialHandlerLDLocal & IDIDManager & IKeyManager & IDataStore & IDataStoreORM & IResolver
+    >({
+      plugins: [
+        new BlsKeyManager({
+          store: new KeyStore(connection),
+          kms: {
+            local: new BlsKeyManagementSystem(privateKeyStore),
+          },
+        }),
+        new DIDManager({
+          store: new DIDStore(connection),
+          defaultProvider: 'did:key',
+          providers: {
+            'did:key': new BlsKeyDidProvider({
+              defaultKms: 'local',
+            }),
+            'did:web': new WebDIDProvider({
+              defaultKms: 'local',
+            }),
+            'did:factom': new FactomDIDProvider({
```

Comment:
Please remove the providers not being used in the example. Main reason is that I don't want our internal registrar links to end up all over our repos until VDX is in production

Reply:
Removed